src/main/java/com/sawyereffect/steps/StartingSteps.java

```diff
@@ -14,22 +9,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
 import java.util.concurrent.TimeUnit;
 
-public class StartingSteps {
+public class StartingSteps extends DriverFactory{
```

Comment:
But why it's better extending than creating the driver? My concern is that StartingSteps is not a DriverFactory and by creating a hierarchy we are indicating that.